dartsim/src/EntityManagementFeatures.cc

```diff
@@ -304,25 +307,29 @@ Identity EntityManagementFeatures::GetLink(
 Identity EntityManagementFeatures::GetLink(
     const Identity &_modelID, const std::string &_linkName) const
 {
-  DartBodyNode *const bn =
-      this->ReferenceInterface<ModelInfo>(_modelID)->model->getBodyNode(
-          _linkName);
-
-  // If the link doesn't exist in "links", it means the containing entity has
-  // been removed.
-  if (this->links.HasEntity(bn))
-  {
-    const std::size_t linkID = this->links.IdentityOf(bn);
-    return this->GenerateIdentity(linkID, this->links.at(linkID));
-  }
-  else
+  auto modelInfo = this->ReferenceInterface<ModelInfo>(_modelID);
```

Comment:
```suggestion
  const auto modelInfo = this->ReferenceInterface<ModelInfo>(_modelID);
```
Based on how I'm seeing this variable being used in the rest of this method, I believe this variable can be `const`.

Reply:
You're right, I changed it to a const ref in https://github.com/ignitionrobotics/ign-physics/pull/227/commits/16da5e14a35a9e3e437287a576e11f014425fe91